test/integration/push_test.rb

```diff
@@ -439,6 +439,26 @@ class PushTest < ActionDispatch::IntegrationTest
 
       assert_response :forbidden
     end
+
+    should "fail when spec.date cannot Marshal.dump" do
```

Comment:
Is this test supposed to be part of this PR?

Reply:
yeah, was part of me trying to get the diff coverage to 100%, since I touched some existing uncovered lines in pusher